doc/changelog/02-specification-language/10076-not-canonical-projection.rst

```diff
@@ -0,0 +1,4 @@
+- Record fields can be annotated to prevent them from being used as canonical projections;
+  see :ref:`canonicalstructures` for details
+  (#10076 <https://github.com/coq/coq/pull/10076>,
```

Comment:
```suggestion
  (`#10076 <https://github.com/coq/coq/pull/10076>`_,
```

Sorry, rst syntax is sometimes quite weird.

Reply:
Can you please explain what it does change? Thank you.

Reply:
See the difference between the entry you added and the other at https://coq.gitlab.io/-/coq/-/jobs/209600067/artifacts/_install_ci/share/doc/coq/sphinx/html/changes.html.

Reply:
Spotted! Thanks.